lib/skiplist.c

```diff
@@ -74,7 +74,7 @@ DEFINE_MTYPE_STATIC(LIB, SKIP_LIST_NODE, "Skip Node")
 
 static int randomsLeft;
 static int randomBits;
-static struct skiplist *skiplist_last_created; /* debugging hack */
+static struct skiplist *skiplist_last_created = NULL; /* debugging hack */
```

Comment:
we're trying not to initialize statics to NULL - I think this will generate a style warning from the CI

Reply:
thanks for the review, initialization removed